docusaurus.config.js

```diff
@@ -4,9 +4,9 @@ module.exports = {
   title: "Camunda Platform 8 Docs",
   tagline: "Documentation for all components of Camunda Platform 8",
   // url: "https://camunda-cloud.github.io",
-  url: "https://docs.camunda.io",
+  url: "https://unsupported.docs.camunda.io",
```

Comment:
I actually don't know what this URL is used for. I think it might be the sitemap; though we aren't generating a sitemap, so I don't think we see this show up anywhere.